recipes/scikit-matter/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "skmatter" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b7a2042d116a5395f5ba81c1efc03df5263b409e36392ee35eefef74857546eb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - scikit-learn >=1.1.0
```

Upstream also lists numpy (and uses it directly):
```suggestion
    - python >=3.8
    - numpy
    - scikit-learn >=1.1.0
```